RELEASE/scripts/autoscend/auto_acquire.ash

```diff
@@ -194,6 +194,10 @@ boolean pullXWhenHaveY(item it, int howMany, int whenHave)
 	{
 		return false;
 	}
+	if(in_lol())
+	{
+		// todo - add check if it is pullable and create error log if it is not
```

Comment:
I think this pr is fine if this todo is addressed.

Reply:
Did it plus a few other things